action.yml

```diff
@@ -0,0 +1,11 @@
+# action.yml
+name: 'NLU Hyperopt'
+description: 'Find optimal hyperparameters for NLU model'
+outputs:
+  best-target-metric:
+    description: 'best intent target metric found'
```

Comment:
I believe the action doesn't have any output, right? This having an output would require some sort of `echo` at the end or at least a print with a specific syntax (never done it myself). You can try by adding another step to the workflow and see if you can access the results of the previous step.

Reply:
Yeah, the output was there before so I left it but i was wondering if it was useless. I'll see if I can `echo` it out